api/v1/paths/developers.js

```diff
@@ -17,6 +18,31 @@ const get = async (req, res) => {
   }
 };
 
+/**
+ * @summary Post developers
+ * @returns {Promise<Object[]>} Promise object represents the newly created developer record
+ */
+const post = async (req, res) => {
+  try {
+    // Check that a body is present in the request
+    if (_.isEmpty(req.body)) {
+      errorBuilder(res, 400, ['No body in request.']);
+    } else {
+      // parse body JSON and check that required attributes are present
+      const { attributes } = JSON.parse(Object.keys(req.body)[0]).data;
+      if (!attributes.name || !attributes.website) {
+        errorBuilder(res, 400, ['Malformed body.']);
+      } else {
+        const result = await developersDao.postDeveloper(attributes);
+        res.status(201).send(result);
+      }
+    }
```

Comment:
As discussed, this part should be pretty simple as [here](https://github.com/osu-mist/notes-api/blob/master/api/v1/paths/notes.js#L23-L30) if the openfile file set correctly. Can you modify this as well?

Reply:
Only thing I needed to add was a check that the body isn't empty. Otherwise a 500 error was returned.